literature.bib

```diff
@@ -15503,6 +15560,19 @@ @inproceedings{STKS:VaMoS12
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2011 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{NNNN:ASE11,
+  author = {Hoan Anh Nguyen and Tung Thanh Nguyen and Hung Viet Nguyen and Nguyen, Tien N.},
+  title = {{iDiff: Interaction-Based Program Differencing Tool}},
+  year = 2011,
+  isbn = {9781457716386},
+  publisher = IEEE,
+  address = USA,
```

Comment:
USA looks weird, is this intended?

Reply:
I guess it should be `Washington`. At least that's the address for the other `ASE11` entries.